dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -4,21 +4,34 @@
 class Api::V1::MlModelsController < ApplicationController
   skip_before_action :verify_authenticity_token
```

Comment:
Is there a chance we want to verify a token for upload?

Reply:
I think ultimately, yes, but since we skip the authenticity token verification in other places, how do you feel about me getting this in to keep us unblocked for end-of-week goals  and circling back to threading the token through?

Reply:
Tracking follow-up work in [AI-76](https://codedotorg.atlassian.net/browse/AI-76)